server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel4.java

```diff
@@ -415,6 +376,7 @@ protected void configureLevel() {
       QProfileBackuper.class,
       QProfileReset.class,
       RubyQProfileActivityService.class,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "RubyQProfileActivityService"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)